Cargo.toml

```diff
@@ -20,7 +23,7 @@ debug = true
 inherits = "release"
 debug = true
 debug-assertions = false
-lto = true
+lto = "thin"
```

Comment:
rhi thicc lto

Reply:
From what I can tell thin and fat lto perform about the same, but thin lto does lto in parallel. But just to be safe, release builds do fat lto.